src/app/pages/account-payment/account-payment/account-payment.component.html

```diff
@@ -46,6 +46,8 @@ <h4>
       </a>
     </div>
     <p>{{ pi.accountIdentifier }}</p>
-    <a *ngIf="!isPreferred" (click)="setAsDefaultPayment(pi.id)">{{ 'account.payment.preferred.link' | translate }}</a>
+    <a *ngIf="!isPreferred" (click)="setAsDefaultPayment(pi.id)">{{ 'account.payment.preferred.link' | translate }}</a
+    >&nbsp;&nbsp;&nbsp;
```

Comment:
if you need space, you bootstrap classes but this should not be necessary here (see next comment)

Reply:
Done.